### PR TITLE
Disable replacing ffmpeg for electron

### PR DESCRIPTION
**Pull request by evidolob**
To restore build on CI we need to disable replacing ffmpeg for electron

### CODE REVIEW

File: dockerfiles/theia-dev/Dockerfile [hunk 1]
```diff
@@ -66,8 +67,8 @@ RUN npm config set prefix "${HOME}/.npm-global" && \
     echo '{"optOut": true}' > ${HOME}/.config/insight-nodejs/insight-yo.json && \
     # Change permissions to let any arbitrary user
     for f in "${HOME}" "/etc/passwd" "/etc/group" "/projects"; do \
-        echo "Changing permissions on ${f}" && chgrp -R 0 ${f} && \
-        chmod -R g+rwX ${f}; \
+    echo "Changing permissions on ${f}" && chgrp -R 0 ${f} && \
```
Comment (sunix) [inline dockerfiles/theia-dev/Dockerfile]: why this change ?

Comment (mmorhun) [inline dockerfiles/theia-dev/Dockerfile]: I think we should have an indent here.
